src/helpers/util.ts

```diff
@@ -13,5 +13,25 @@ const getRoleLabel = (roleName: string) => {
             return APP.ROLE_LABEL.SUPER_ADMIN;
     }
 }
-
-export default { getRoleLabel };
\ No newline at end of file
+// Function to format the status label
+export const formatLabel = (labelStatus: string) => {
+    let lowerLabel = labelStatus.toLowerCase();
+    lowerLabel = lowerLabel.charAt(0).toUpperCase() + lowerLabel.slice(1);
+    switch (lowerLabel) {
+      case "Underreview":
```

Comment:
@BharathTarento Also create METRIC_Label_KEY with all the keys and use that here for case match.